pkg/agent/util/net_windows.go

```diff
@@ -0,0 +1,215 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
```

Comment:
@tnqn: do you think we can mvoe files in util/ethtool/ and net_os.go to a single pkg util/net/?

Reply:
Yes, they sounds close.

Reply:
I now feel stronger we should put net_os.go to util/net, and maybe move ethtool files there too. We added more and more funcs here.

Reply:
Yes, I plan to move them after this PR and policy only PR is checked in, don't want to make them have to solve conflicts again.

Reply:
Sounds good.